docs/static/settings-file.asciidoc

```diff
@@ -28,4 +29,82 @@ pipeline.batch.size: 125
 pipeline.batch.delay: 5
 -------------------------------------------------------------------------------------
 
-See <<command-line-flags>> for a description of the available options.
+The `logstash.yml` file includes the following settings:
+
+ *`node.name`*::
```

Comment:
Yeah, I think a table will be easy to read. Can we try that? +1 on Setting, Description, and Default value as column heading.s